config/config.go

```diff
@@ -11,12 +11,20 @@ import (
 	"regexp"
 
 	"github.com/mozilla-services/yaml"
+	"github.com/sirupsen/logrus"
 	"go.mozilla.org/sops"
 	"go.mozilla.org/sops/gcpkms"
 	"go.mozilla.org/sops/kms"
+	"go.mozilla.org/sops/logging"
 	"go.mozilla.org/sops/pgp"
 )
 
+var log *logrus.Logger
+
+func init() {
+	log = logging.NewLogger("CONFIG")
+}
+
```

Comment:
@autrilla something like this?

Reply:
Yes, that's exactly what I was looking for